kits/igor/igor2DemoIntegrated.m

```diff
@@ -831,9 +877,9 @@ function igor2DemoIntegrated( cam_module )
         cmdChassisVel = cmdVel;
 
         % PID Controller that set a desired lean angle
-        velP = 5 / .33;
-        velI = 3 / 30;
-        velD = .3 / 1;
+        velP = 20;
+        velI = 3 * (latestPhoneIO.a5+1);  % Scale I term with joystick
+        velD = 0.5;
```

Comment:
@rLinks234 we should also change this (and changes on lines 892 and 935) in the Python API Igor example.  This improves performance with new pose filter.

Reply:
Agreed. I'll make a note to do it